v2/example_test.go

```diff
@@ -61,7 +62,7 @@ func ExampleWrapV1File() {
 	if err != nil {
 		panic(err)
 	}
-	fmt.Println(bs.Get(roots[0]))
+	fmt.Println(bs.Get(context.Background(), roots[0]))
```

Comment:
for examples, I wonder if we should use `context.TODO()` instead, to highlight to the user that they should probably replace the context with something more useful - or consciously use a background context.

I guess it's not super important here, since the CAR blockstore doesn't yet use the context at all, but it might in the future.